### DISCUSSION TITLE
Force dark mode on a specific website

### DISCUSSION THREAD

**Discussion by Mutahhar** [darkreader/darkreader#9875] [Q&A]
I have set the DarkReader to auto mode (Switches according to system dark mode). My system's mode is light. 
I want to force dark mode on a website like GitHub, regardless of my system's mode.
I tried selecting "Theme for github.com" (shown below in screenshot) and changing the `Scheme` value to `Dark` but it is not working. The website is still showing in light mode. 
NOTE: github.com is an example I tried with other websites but the same behavior.

![image](https://user-images.githubusercontent.com/11645672/191255673-81bd4af8-12ae-4595-8b5a-1a3ada0fe3f0.png)

**Comment by Gusted** [comment]
That's not possible, the On/automation/Off mode is global and not overrideable.

**Comment by Mutahhar** [comment]
@Gusted 
Is it possible to request it as a feature?

**Reply by Gusted** [reply]
No, I wouldn't consider exemptions to the global toggle a great feature to be added. As that will increase complexity (code and UI wise) and people *might* forget about these and open issues why the on/off toggle doesn't work anymore.

**Reply by Gedof** [reply]
I was also looking for something like this. I would like to switch back and forth on most websites, but would like to have some exceptions to choose whether they are always light or dark. Probably won't happen, but just commenting to add a voice to the idea in case it's ever relevant.
